matrix.yaml

```diff
@@ -0,0 +1,16 @@
+CUDA_VER:
+  - "11.2.2"
+  - "11.8.0"
+PYTHON_VER:
+  - "3.9"
+  - "3.10"
+LINUX_VER:
+  - "ubuntu20.04"
+  - "ubuntu22.04"
```

Comment:
since `LINUX_VER` is now a computed property and not something that we explicitly paramaterize, what do you think about moving it to the `compute-matrix.jq` function?

This follows what we do for the `ARCHES` values. It would also allow us to clean this file up a bit.

Reply:
Good idea!

Reply:
> Would be good to remove `DASK_SQL_VER` too since it's not really a matrix parameter. But that can be a future enhancement I think.

I was thinking the same.

but then I noticed that the Dockerfile already has the version included here: https://github.com/raydouglass/docker/blob/32fa035357fa0f0603e7c24861f08000cc0dd8b4/Dockerfile#L8.

So it should be a no-op to just drop it from the matrix and associated GHAs workflows now and use that Dockerfile `ARG` version as the source of truth, right?

Reply:
I think the `update-dask-sql.yml` workflow would need to be updated too. But that whole workflow should probably be replaced by renovate which is a bigger job.